### PR TITLE
Feature: Restart command

### LINKED ISSUE #109: add 'restart' command
A lot of developers are asking how to "restart" their application.  Adding a `restart` command could update a dummy environment variable causing k8s to do a new rolling deployment.

```
$ harbor-compose restart
```

### PR DESCRIPTION
**Pull request by jkurz**
* add restart function
* closes #109

### CODE REVIEW

File: cmd/harborAPI.go [hunk 1]
```diff
@@ -548,3 +549,41 @@ func CatalogCustoms(shipment string, env string, buildToken string, catalogReque
...
+	values := make(map[string]interface{})
+	values["shipment"] = shipment
+	values["env"] = env
+	template, _ := uritemplates.Parse(config.ShipitURI + "/v1/shipment/{shipment}/environment/{env}/envVar/RESTART")
+	uri, _ := template.Expand(values)
+	if Verbose {
+		log.Printf("deleting: " + uri)
+	}
+
+	res, _, _ := delete(username, token, uri)
```
Comment (jritsema) [inline cmd/harborAPI.go:570]: why not get/put/post rather than delete/add here for a cleaner audit trail?

File: cmd/restart.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+func restart(cmd *cobra.Command, args []string) {
+
+	//make sure user is authenticated
+	username, token, err := Login()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	//read the harbor compose file
+	harborCompose := DeserializeHarborCompose(HarborComposeFile)
```
Comment (jritsema) [inline cmd/restart.go:32]: You can replace the 2 calls to unmarshal the compose files with a call to `unmarshalCompose` which returns both.

**Review by jritsema — CHANGES_REQUESTED** [review]

File: cmd/harborAPI.go (hunk 1 above)
Comment (jkurz) [reply cmd/harborAPI.go:570]: just following current logic. Once shipit bulk is in, we can make this much simpler. Also, you can't get an envVar currently. I could just write this to use the bulk api, but that probably should all be done at once, so things are clean.

Comment (jritsema) [reply cmd/harborAPI.go:570]: Oh ya, forgot you can't GET envvars.

File: cmd/restart.go (hunk 2 above)
Comment (jritsema) [reply cmd/restart.go:32]: Oops, `unmarshalCompose` takes yaml vs files (we need to support that).

File: cmd/restart.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+			ComposeFiles: []string{DockerComposeFile},
+		},
+	}, nil)
+
+	if err != nil {
+		log.Fatal("error parsing compose file" + err.Error())
+	}
+
+	//validate the compose file
+	_, err = dockerComposeProject.Config()
```
Comment (jritsema) [inline cmd/restart.go]: For now, you can at least replace this with a call to `DeserializeDockerCompose`.

File: cmd/restart.go (hunk 2 above)
Comment (jkurz) [reply cmd/restart.go:32]: updated to just get HarborComposeYaml, since that is all we actually need.

**Comment by jritsema** [comment]
LGTM

**Review by jritsema — COMMENTED** [review]
LGTM

**Pull request merged**